loop-de-loop.ipynb

```diff
@@ -5,7 +5,7 @@
    "id": "a57532ae-cd7d-4b2a-9edb-eb83a8f3a09b",
```

Comment:
<p>Yes, that was unintentional</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/loop-de-loop/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>